lms/static/js/instructor_dashboard/membership.js

```diff
@@ -153,28 +160,49 @@ such that the value can be defined later than this assignment (file load order).
                         class: 'revoke'
                     });
                     $revokeBtn.click(function() {
-                        return authlistwidgetreloadlist.modify_member_access(member.email, 'revoke', function(err) {
+                        authlistwidgetreloadlist.modify_member_access(member.email, 'revoke', function(err) {
                             if (err !== null) {
-                                return authlistwidgetreloadlist.show_errors(err);
+                                authlistwidgetreloadlist.show_errors(err);
                             }
                             authlistwidgetreloadlist.clear_errors();
-                            return authlistwidgetreloadlist.reload_list();
+                            authlistwidgetreloadlist.reload_list();
                         });
                     });
-                    return authlistwidgetreloadlist.add_row([member.username, member.email, $revokeBtn]);
+                    if (authlistwidgetreloadlist.rolename === 'Group Moderator') {
```

Comment:
We discussed test coverage some. If feasible, a JS unit test covering the logic of this block of code (171-192) would be great.